lib/gems/pending/appliance_console/cli.rb

```diff
@@ -70,6 +70,14 @@ def tmp_disk?
       options[:tmpdisk]
     end
 
+    def date_time?
```

Comment:
why set?  this function is to check if there is the option datetime no for set it